src/ddb/ddb_common.h

```diff
@@ -42,92 +42,46 @@
 		printf(fmt, __VA_ARGS__); \
 	} while (0)
 
-struct dv_tree_path {
-	uuid_t		vtp_cont;
-	daos_unit_oid_t vtp_oid;
-	daos_key_t	vtp_dkey;
-	daos_key_t	vtp_akey;
-	daos_recx_t	vtp_recx;
-	bool		vtp_is_recx;
-};
-
-/* Is used while parsing user input for building the vos tree path. The builder can use branch
- * indexes that will be converted to the appropriate vos part (i.e. cont uuid, object id, etc).
- */
-struct dv_tree_path_builder {
-	daos_handle_t		 vtp_poh; /* pool handle */
-	struct dv_tree_path	 vtp_path;
-
-	/*
-	 * If a key value is passed instead of an index, then need a buffer to copy the key value
-	 * into.
-	 */
-	uint8_t			*vtp_dkey_buf;
-	uint8_t			*vtp_akey_buf;
-
-	/* Used during the verification process */
-	uint32_t		 vtp_current_idx;
-	/*
-	 * A user can pass an index of the path part. These indexes will be used to complete
-	 * the path parts.
-	 */
-	uint32_t		 vtp_cont_idx;
-	bool			 vtp_cont_verified;
-	uint32_t		 vtp_oid_idx;
-	bool			 vtp_oid_verified;
-	uint32_t		 vtp_dkey_idx;
-	bool			 vtp_dkey_verified;
-	uint32_t		 vtp_akey_idx;
-	bool			 vtp_akey_verified;
-	uint32_t		 vtp_recx_idx;
-	bool			 vtp_recx_verified;
-};
-
-static inline bool
-dv_has_cont(struct dv_tree_path *vtp)
-{
-	return !uuid_is_null(vtp->vtp_cont);
-}
-
-static inline bool
-dv_has_obj(struct dv_tree_path *vtp)
-{
-	return !(vtp->vtp_oid.id_pub.lo == 0 &&
-		 vtp->vtp_oid.id_pub.hi == 0);
-}
-
-static inline bool
-dv_has_dkey(struct dv_tree_path *vtp)
-{
-	return vtp->vtp_dkey.iov_len > 0;
-}
-
-static inline bool
-dv_has_akey(struct dv_tree_path *vtp)
-{
-	return vtp->vtp_akey.iov_len > 0;
-}
-
-static inline bool
-dv_has_recx(struct dv_tree_path *vtp)
-{
-	return vtp->vtp_recx.rx_nr > 0;
-}
-
-static inline bool
-dvp_is_complete(struct dv_tree_path *vtp)
-{
-	return dv_has_cont(vtp) && dv_has_obj(vtp) && dv_has_dkey(vtp) && dv_has_akey(vtp);
-}
-
-static inline bool
-dvp_is_empty(struct dv_tree_path *vtp)
-{
-	return !dv_has_cont(vtp) && !dv_has_obj(vtp) && !dv_has_dkey(vtp) && !dv_has_akey(vtp);
-}
-
-void
-vtp_print(struct ddb_ctx *ctx, struct dv_tree_path *vt_path, bool include_new_line);
+/* [todo-ryon]: clean this up */
+///* Is used while parsing user input for building the vos tree path. The builder can use branch
+// * indexes that will be converted to the appropriate vos part (i.e. cont uuid, object id, etc).
+// */
+//struct dv_tree_path_builder {
+//	daos_handle_t		 vtp_poh; /* pool handle */
+//	struct dv_tree_path	 vtp_path;
+//
+//	/*
+//	 * If a key value is passed instead of an index, then need a buffer to copy the key value
+//	 * into.
+//	 */
+//	uint8_t			*vtp_dkey_buf;
+//	uint8_t			*vtp_akey_buf;
+//
+//	/* Used during the verification process */
+//	uint32_t		 vtp_current_idx;
+//	/*
+//	 * A user can pass an index of the path part. These indexes will be used to complete
+//	 * the path parts.
+//	 */
```

Comment:
(style)  Block comments use a trailing */ on a separate line
(style)  do not use C99 // comments